app/src/main/java/org/ometa/lovemonster/ui/activities/LoveListActivity.java

```diff
@@ -57,7 +61,49 @@ public void onSuccess() {
     @Override
     public boolean onCreateOptionsMenu(final Menu menu) {
         getMenuInflater().inflate(R.menu.menu_love_list, menu);
-        setUserProfileIcon(menu.getItem(0), 2);
+
+        MenuItem searchItem = menu.findItem(R.id.action_search);
+        SearchView searchView = (SearchView) searchItem.getActionView();
+        searchView.setQueryHint("Username...");
+        searchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
+            @Override
+            public boolean onQueryTextSubmit(final String query) {
+                logger.debug("Search for " + query);
+                LoveMonsterClient client = LoveMonsterClient.getInstance();
```

Comment:
I'm thinking about adding a boolean instance variable here to track if there's a request in flight, and not submitting if we're already searching.